Makefile

```
@@ -1,25 +1,40 @@
-.PHONY: dev dev-backend dev-frontend test lint build
+.PHONY: install dev dev-backend dev-frontend test lint typecheck build clean
+
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)